99-x52pro.rules

```diff
@@ -1,9 +1,9 @@
 #X52 Flight System
-ACTION=="add", SUBSYSTEMS=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0255", MODE="660", GROUP="plugdev"
-ACTION=="add", SUBSYSTEMS=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="075c", MODE="660", GROUP="plugdev"
+ACTION=="add", SUBSYSTEMS=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0255", TAG+="uaccess"
```

Comment:
According to [Allowing regular users to use devices](https://wiki.archlinux.org/title/Udev#Allowing_regular_users_to_use_devices) this does only work if the rule file lexically precedes "/lib/udev/rules.d/73-seat-late.rules", so "99" is "too late".